velox/buffer/tests/BufferTest.cpp

```diff
@@ -40,6 +40,7 @@ class BufferTest : public testing::Test {
  protected:
   static void SetUpTestCase() {
     FLAGS_velox_enable_memory_usage_track_in_default_memory_pool = true;
+    config::translateFlagsToGlobalConfig();
```

Comment:
For unit test we can just use `globalConfig`

Reply:
I did it this way so that `config::translateFlagsToGlobalConfig()` also gets tested.